api/src/main/java/com/codeforcommunity/dto/report/ReportWithoutLibrary.java

```diff
@@ -0,0 +1,41 @@
+package com.codeforcommunity.dto.report;
+
+public class ReportWithoutLibrary extends ReportGeneric {
+
+  private boolean wantsLibrary;
```

Comment:
For the sake of consistency, can we make `wantsLibrary` and `hasSpace` type `Boolean` instead of `boolean`?